sonar-plugin-api/src/main/java/org/sonar/api/issue/Issue.java

```diff
@@ -139,8 +139,9 @@
   String resolution();
 
   /**
-   * Login of the user who reported this issue. Null if the issue is reported by a rule engine.
+   * @deprecated since 5.5, manual issue feature has been dropped.
    */
+  @Deprecated
   @CheckForNull
   String reporter();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)